internal/test_helpers/course_definition.yml

```
@@ -1280,4 +1285,39 @@ stages:
+    name: "Limiting history entries"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for `history N` to print the last N commands.
+
+  - slug: "rh7"
+    primary_extension_slug: "history"
+    name: "Up arrow navigation"
+    difficulty: medium
+    description_md: |-
+      hi
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)